Improvements/more documentation (#38)
hermeznetwork/price-updater-service: README.md

## Usage 

> The system is designed to execute the commands listed above. The commands are listed to be executed in the order below.

Author: ```suggestion
The system has been designed to run through the commands listed above. The commands are listed to be executed in the order below.
```

There are two commands related to providers:

> 1. [REQUIRED] `$ priceupdater change-prirotiry --provider <provider-name-1>,<provider-name-2>,<provider-name-3>`

Author: ```suggestion
1. [REQUIRED] `$ priceupdater change-prirotiry --priority <provider-name-1>,<provider-name-2>,<provider-name-3>`
```
